### PR TITLE
xds: add config for least_request LB policy extension

### PR DESCRIPTION
**Pull request by temawi**
Signed-off-by: Terry Wilson <tmwilson@google.com>

Commit Message: xds: add config for least_request LB policy extension
Additional Description: Adds extension config for LEAST_REQUEST policy, similar to what was done in https://github.com/envoyproxy/envoy/pull/19517 and https://github.com/envoyproxy/envoy/pull/20423
Risk Level: Low
Testing: N/A
Docs Changes: Included in PR
Release Notes: N/A
Platform Specific Features: N/A

### CODE REVIEW

File: api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/least_request/v3;least_requestv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Least Request Load Balancing Policy]
+
+// This configuration allows the built-in LEAST_REQUEST LB policy to be configured via the LB policy
+// extension point. See the :ref:`load balancing architecture overview
+// <arch_overview_load_balancing_types>` for more information.
+// [#extension: envoy.clusters.lb_policy]
+message LeastRequest {
```
Comment (markdroth) [inline api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto:22]: This should also have the [`active_request_bias`](https://github.com/envoyproxy/envoy/blob/f2b3fb39955414720718fdb8ccb4e0fb815523bf/api/envoy/config/cluster/v3/cluster.proto#L420) and [`slow_start_config`](https://github.com/envoyproxy/envoy/blob/f2b3fb39955414720718fdb8ccb4e0fb815523bf/api/envoy/config/cluster/v3/cluster.proto#L424) fields.

**Review by markdroth — COMMENTED** [review]
This is another small piece of #20634.

File: api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto (hunk 1 above)
Comment (temawi) [reply api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto:25]: I don't know how I missed those. Done.

**Comment by markdroth** [comment]
Hmm, that didn't work for some reason.

@mattklein123 or @lizan, can one of you please provide a non-Google review and get this merged?  Thanks!

**Review by mattklein123 — APPROVED** [review]

**Pull request merged**